include/networkit/Globals.hpp

```diff
@@ -23,6 +23,14 @@ using omp_index = int64_t;
 using omp_index = index;
 #endif // _MSC_VER
 
+#ifdef NDEBUG
+#define DEBUG_ONLY(x)
+#else
+#define DEBUG_ONLY(x) = x;
+#endif
```

Comment:
This should be called `NETWORKIT_DEBUG_ONLY` to keep the macro namespace clean. The `!NDEBUG` case also seems to be broken (what is the `=` for?).

Reply:
I typed this from memory since it was only a suggestion for now.